topics/wallet-iteration-standard/ledger-extension.md

```diff
@@ -0,0 +1,91 @@
+# ICRC-27: Ledger extension for Wallet Interaction Standard
+
+| Status |
+|:------:|
+| Draft  |
+
+## Summary
+
+This specification describes a ledger extension for the ICRC-25 Wallet Interaction Standard.
+
+## Terminology
+
+* wallet: A service that manages a user's keys and can sign and perform canister calls on their behalf.
+* relying party: A service that wants to request calls on a specific canister.
+
+## Messages
+
+### `permission`
+
+Additional `ledger` scope is passed in the request and in the response an additional `subaccounts` property is returned in each `identity` when the wallet supports this extension.
+
+There are many types of wallets:
+- Wallets that support many different standards
+- Wallets that support multiple networks
+- Wallets that keep a list of subaccounts per ledger
+- Wallets that keep a list of subaccounts used across all ledgers
+
+To support all these different wallets and pass these details to the relying party, the subaccounts in this spec are defined with multiple properties to indicate on which networks, standards and ledgers the subaccounts are meant to be used.
+
+#### Request
+
+The `ledger` scope is passed to the ICRC-25 `permission` request message scopes, no additional fields in the request are required in this spec.
+
+#### Response
+
+`identities`: Same as ICRC-25 response with additional `ledger` field.
```

Comment:
For a better structure overview, I would prefer to see `ledger` listed as a bullet point, something like:

> `identities`: ICRC-25 response `identities` extended with additional fields.
>  - `ledger`
>    - `subaccounts` ...

otherwise it's not that clear that the `subaccounts` field belongs to `ledger` (at least I needed a moment to get it 😅 )

Reply:
Good point! I've added it as bullet point.